custom_components/elegoo_printer/elegoo_sdcp/elegoo_printer.py

```
@@ -10,8 +10,13 @@
 
 import websocket
 
+from custom_components.elegoo_printer.elegoo_sdcp.models.print_history_detail import (
+    PrintHistoryDetail,
+)
+
 from .const import DEBUG
 from .models.attributes import PrinterAttributes
+from .models.print_history_detail import PrintHistoryDetail
```

Ruff (0.8.2)

Redefinition of unused `PrintHistoryDetail` from line 14

(F811)